service-deployment/bootstrap/prometheus/start.sh

```diff
@@ -21,11 +21,14 @@
 
 pushd $(dirname "$0") > /dev/null
 
-NAMESPACE=${NAMESPACE:-monitor}
```

Comment:
please also remove name space here: [https://github.com/Microsoft/pai/blob/master/service-deployment/bootstrap/prometheus/start.sh#L24-L26](https://github.com/Microsoft/pai/blob/master/service-deployment/bootstrap/prometheus/start.sh#L24-L26)

Reply:
Solved

Reply:
you didn't include it in this PR

---
In reply to: [158423898](https://github.com/Microsoft/pai/pull/119#discussion_r158423898) [](ancestors = 158423898)